index.md

```diff
@@ -1,8 +1,15 @@
 # ПЕРЕД УХОДОМ НЕ ЗАБЫВАЕМ
 
+## ПРОВЕРИТЬ
+
+1. НАЛИЧИЕ ЛЮДЕЙ В ПОМЕЩЕНИЯХ
+1. НАЛИЧИЕ КЛЮЧА ОТ ТУАЛЕТА НА ТАРЕЛОЧКЕ
+1. ДВЕРЦУ ХОЛОДИЛЬНИКА
```

Comment:
We probably should check something more specific than just presence of the door

Reply:
Looks like redundant action, because the door is always there. I really don't want to waste my time doing such work in the evening